src/views/virtualmachines/list/hooks/constants.ts

```diff
@@ -16,8 +16,8 @@ export const getVMListQueries = (namespace: string) => {
 
   return {
     [VMListQueries.CPU_REQUESTED]: `kube_pod_resource_request{resource='cpu',${namespaceFilter}}`,
-    [VMListQueries.CPU_USAGE]: `rate(kubevirt_vmi_cpu_usage_seconds_total{${namespaceFilter}}[5m])`,
+    [VMListQueries.CPU_USAGE]: `rate(kubevirt_vmi_cpu_usage_seconds_total{${namespaceFilter}}[30s])`,
```

Comment:
IMHO changing the window to 30s impacts functionality - we should double check if that's OK

Reply:
I think 30s is right as this is the minimum promethus is getting updates , and we are showing it on the VM list, without any way to change values from the UI, so it will always show the latest updated value which is what we want , the only concern is the amount of rerender we will get.. @upalatucci is it safe for performance?

Reply:
yes we are not changing the number of re-rendering and the pooling timeout. Just the data that we fetch. The pooling timeout will remain the same: `15s` ( the default one)